doc/how_to_guides/isaac_panda/.docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM osrf/ros:humble-desktop-jammy
+
+SHELL ["/bin/bash", "-c", "-o", "pipefail"]
+
+RUN echo "deb [trusted=yes] https://raw.githubusercontent.com/moveit/moveit2_packages/jammy-humble/ ./" \
+    | sudo tee /etc/apt/sources.list.d/moveit_moveit2_packages.list
+RUN echo "yaml https://raw.githubusercontent.com/moveit/moveit2_packages/jammy-humble/local.yaml humble" \
+    | sudo tee /etc/ros/rosdep/sources.list.d/1-moveit_moveit2_packages.list
+
+# Bring the container up to date to get the latest ROS2 humble sync on 01/27/23
+# hadolint ignore=DL3008, DL3013
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update && apt-get upgrade -y && rosdep update
+
+# Install packages required to run the demo
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get install -y --no-install-recommends \
+    ros-humble-moveit \
+    ros-humble-moveit-resources
+
+# Create Colcon workspace and clone the needed source code to run the demo
+RUN mkdir -p /root/isaac_moveit_tutorial_ws/src
+WORKDIR /root/isaac_moveit_tutorial_ws/src
+RUN git clone https://github.com/PickNikRobotics/isaac_ros2_control.git
+# Docker does not allow copying in parent directories from the host so grab another copy of the tutorials
+RUN git clone https://github.com/MarqRazz/moveit2_tutorials.git -b pr-isaac_tutorial --depth 1
```

Comment:
I was going to say that this is awkward for this reason, meaning that the copy that the user just cloned and possibly modified is different from the one that is being pulled here.

The trick is to change the context, which in the Dockerfile is shown here and then you can use the `COPY` directive. For example,

```
    build:
      context: ../../..
      dockerfile: doc/how_to_guides/isaac_panda/.docker/Dockerfile
```

But then you have to also add a `.dockerignore` file or a bunch of not desired things will copy over.

Reply:
I would prefer we `apt install ros-humble-moveit2-tutorials` and remove the need to build the package completely in the container! @tylerjw do you have thoughts on me doing this before we merge? 

I would also like to get `isaac_ros2_control` from binary if that were possible, but that is going to take some time for @JafarAbdi to wrap that up.